frenchrubyshops.csv

```diff
@@ -43,3 +43,4 @@ Vodeclic,http://vodeclic.com/,product,Levallois
 Weroom,https://www.weroom.com/,product,Paris
 Wizypay,http://wizypay.com/,product,Paris
 Yespark,https://www.yespark.fr/,product,Paris
+Capsens,http://www.capsens.eu,crowdfunding, Paris
```

Comment:
Should be `product` instead of `crowdfounding`.
And you have a useless extra white space before `Paris` :wink: